jobs/build/ocp/Jenkinsfile

```diff
@@ -642,11 +655,68 @@ Please direct any questsions to the Continuous Delivery team (#aos-cd-team on IR
                 sh "ose_images.sh --user ocp-build build_container --branch rhaos-${BUILD_VERSION}-rhel-7 --group base --repo https://raw.githubusercontent.com/openshift/aos-cd-jobs/master/build-scripts/repo-conf/aos-unsigned-building.repo"
             }
 
-            buildlib.oit """
+            waitUntil {
+              try {
+                exclude = ""
+                if ( BUILD_EXCLUSIONS != "" ) {
+                  exclude = "-x ${BUILD_EXCLUSIONS} --ignore-missing-base"
```

Comment:
This won't work for space delimited. The second exclude will be treated like an argument unless you wrap it in quotes. I would preprocess BUILD_EXCLUSIONS somewhere near the top of the pipeline to clean it up:
- replace commas with whitespace
- split on whitespace
- join back with comma

This will turn things like "cri-o-docker,    xyz-docker" into parameter safe "cri-o-docker,xyz-docker".